tests/consensus_tests/test_strict_mode.py

```
@@ -245,3 +245,108 @@ def test_write_rate_limiting_across_node(tmp_path: pathlib.Path):
             return
 
     raise AssertionError("rate limiter was never triggered")
+
+
+def test_max_update_by_filter_limit(tmp_path: pathlib.Path):
+    """`max_update_by_filter_limit` is enforced by every shard on the point set
+    its own filter scan resolved to, before anything is written."""
+    peer_urls, peer_dirs, bootstrap_url = start_cluster(tmp_path, N_PEERS)
```

Ruff (0.16.2)

[warning] Unpacked variable `peer_dirs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)